control/strategy/strategy.go

```diff
@@ -17,17 +17,28 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package routing
+package strategy
 
 import (
 	"time"
-)
 
-type SelectablePluginPool interface {
-}
+	"github.com/intelsdi-x/snap/core"
+)
 
 type SelectablePlugin interface {
 	HitCount() int
 	LastHit() time.Time
 	String() string
 }
+
+type RoutingAndCaching interface {
```

Comment:
Do you think there will be other strategy types?

I guess what I'm getting at is why not `strategy.Strategy`?